app.rb

```diff
@@ -0,0 +1,150 @@
+require_relative 'book'
+require_relative 'label'
+require_relative 'music_album'
+require_relative 'genre'
+require_relative 'game'
+require_relative 'author'
+require_relative 'preserve'
+
+require 'date'
+
+class App
+  def initialize
+    @preserve = Preserve.new
+    @preserve.load_music_albums
+    @preserve.load_genres
+    @preserve.load_labels
+    @preserve.load_books
+    @preserve.load_games
+    @preserve.load_authors
+  end
+
+  def list_books
+    return puts 'No books found' if @preserve.books.empty?
+
+    @preserve.books.each_with_index do |book, index|
+      puts "#{index + 1}) Label: #{book.label.title}, Publisher: #{book.publisher}, Publish date: #{book.publish_date}"
+    end
+  end
+
+  def list_labels
+    return puts 'No labels found' if @preserve.labels.empty?
+
+    @preserve.labels.each_with_index do |label, index|
+      puts "#{index + 1}) Title: #{label.title}, Color: #{label.color}"
+    end
+  end
+
+  def list_music_albums
+    return puts 'No music albums found' if @preserve.music_albums.empty?
+
+    @preserve.music_albums.each_with_index do |music_album, index|
+      puts "
+      #{index + 1})  On spotify: #{music_album.on_spotify}
+          Publish date: #{music_album.publish_date}
+      "
+    end
+  end
+
+  def list_genres
+    return puts 'No genres found' if @preserve.genres.empty?
+
+    @preserve.genres.each_with_index do |genre, index|
+      puts "#{index + 1}) Name: #{genre.name}"
+    end
+  end
+
+  def list_games
+    return puts 'No games found' if @preserve.games.empty?
+
+    @preserve.games.each_with_index do |game, index|
+      puts "#{index + 1} Publish Date: #{game.publish_date},
+      Multiplayer: #{game.multiplayer}, Last played at: #{game.last_played_at}"
+    end
+  end
```

Comment:
Whether listing books, music albums, or games, it should display all the information about that particular item. That includes information about the label, author, and genre. However, in your case, it displays very limited information which is not very useful and doesn't give any real information about the item. Kindly fix this issue. Eg., have a look at the screenshot below, when i list music albums.

![image](https://github.com/AbbasSarwar/Ruby-Capstone-Project/assets/33268927/60506295-1114-4ec2-9154-f718ddb17eb1)